modified-openapi.json

```diff
@@ -1794,178 +1651,115 @@
         },
         "oneOf": [
           {
-            "allOf": [
-              {
-                "required": [
-                  "type"
-                ],
-                "type": "object",
-                "properties": {
-                  "type": {
-                    "type": "string",
-                    "pattern": "^[Aa][Dd][Mm][Ii][Nn]$",
-                    "description": "An admin token. Must be the string \"admin\" (not case sensitive).",
-                    "example": "admin"
-                  }
-                }
+            "required": ["type", "tokenName"],
```

Comment:
This is what fixes the schema, removing the nested `allOf` which are inside a `oneOf`